internal/config/config.go

```diff
@@ -123,9 +122,9 @@ var Defaults = func(o *Config) error {
 
 	o.MaxRecordAge = providers.ProvideValidity
 
-	o.BucketSize = defaultBucketSize
-	o.Concurrency = 10
-	o.Resiliency = 3
+	o.BucketSize = amino.DefaultBucketSize
+	o.Concurrency = amino.DefaultConcurrency
+	o.Resiliency = amino.DefaultResiliency
 	o.LookupCheckConcurrency = 256
```

Comment:
Another candidate for defaults?  Also, wish there was some explanation why 256 is the chosen value for concurrency.